critiquebrainz/ws/review/views.py

```diff
@@ -337,6 +337,7 @@ def review_list_handler():
     :query limit: results limit, min is 0, max is 50, default is 50 **(optional)**
     :query offset: result offset, default is 0 **(optional)**
     :query language: language code (ISO 639-1) **(optional)**
+    :query review_type: ``review`` or ``rating`` **(optional)**
```

Comment:
Thanks for updating the documentation for this parameter. I also added some descriptive text in the change that I suggested. We should add this text to this documentation to explain what the parameter does.
I also suggested some improvements to some other parameter documentation for this function that we should improve.